ddtrace/contrib/aiopg/patch.py

```diff
@@ -55,3 +96,94 @@ def _unroll_args(obj, scope=None):
      psycopg2.extensions, 'register_type',
      _extensions_register_type),
 ]
+
+
+@asyncio.coroutine
+def _patched_acquire(acquire_func, instance, args, kwargs):
+    parsed_dsn = psycopg2.extensions.make_dsn(instance._dsn,
```

Comment:
don't we have alternatives here? this drops too many psycopg2 versions that we still need to support :(